src/main/resources/plugin.yml

```diff
@@ -11,9 +11,19 @@ commands:
   sectionon:
     description: Enables automatic teleports between sections for a player
     usage: /sectionon [player]
+    permission: deeperworld.admin
```

Comment:
We do a permission check in code [here](https://github.com/MineInAbyss/DeeperWorld/blob/2a86f54dbefe84dca51ec97d1d0e4baa63b09a00/src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.java#L31), but this is just the permission for moving between layers. I'd rather keep things consistent and just create this admin permission in code and add a check to sectionon/sectionoff.

Perhaps it would have been better to do this in config originally but since I want to start using idofront's command api here once it's ready, it's better to keep it all in code, since idofront handles all the permission stuff on its own without a config.

Reply:
We synced about this in the discord